### DISCUSSION TITLE
The AntiMiroX is not working in game

### DISCUSSION THREAD

**Discussion by cyclone313** [AntiMicroX/antimicrox#547] [Q&A] [answered]
After I set up the mapping with the controller, it is working as I can see I can it is typing in notepad++
However, when I open an online game, the controller has no response. 

Is anybody facing the same issue? Please help.

**Comment by pktiuk** [comment, marked as answer]
Some games, especially the ones using anticheat may not respect inputs from AntiMicroX. (see https://github.com/AntiMicroX/antimicrox/issues/406#issuecomment-1107056079)

**Comment by cyclone313** [comment]
Got it. Thanks for your reply.
